content/en/getting_started/synthetics/api_test.md

```diff
@@ -14,131 +14,128 @@ further_reading:
     - You can select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
-    - You can set environment tags such as `env:prod` and `app:shopist` and additional tags on your test. Tags allow you to stay organized and quickly find test you're interested in on the homepage.
-    - Click **Test URL**.
+    - You can set tags such as `env:prod` and `app:shopist` on your test. Tags allow you to keep your test suite organized and quickly find test you're interested in on the homepage.
+
+6. Click **Test URL** to trigger a sample test run.
 
 {{< img src="getting_started/synthetics/api-test-config-2.jpg" alt="API test configuration"  style="width:60%;">}}
 
-#### Define assertions
+### Define assertions
 
-Datadog automatically populates basic assertions about your endpoint's response. Assertions define the alert condition and are customizable. 
+Clicking **Test URL** automatically populates basic assertions about your endpoint's response. Assertions define what a successful test run is.
 
-In this example, three default assertions populate after the URL is tested:
+In this example, three default assertions populate after triggering the sample test run:
 
 {{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure"  style="width:90%;">}}
 
-To add a custom assertion, click anywhere in the response preview or click **New Assertion**.
+Assertions are fully customizable. To add a custom assertion, click on elements of the response preview or click **New Assertion**. 
 
 {{< img src="getting_started/synthetics/api-test-configuration-2.gif" alt="Browser test failure"  style="width:90%;">}}
 
-#### Select locations 
+### Select locations 
 
 Select one or more **Managed Locations** or **Private Locations** to run your test from.
 
-Your managed locations often include public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select **Private Locations** instead.
+Managed locations allow you to test public-facing websites and endpoints. To test internal applications or simulate user behavior in discrete geographic regions, select one of your **Private Locations** instead.
 
-For more information, see [Getting Started with Private Locations][3].
+For more information on how to set up private locations, see [Getting Started with Private Locations][5].
 
-#### Specify test frequency
+### Specify test frequency
 
 Select the frequency of your test runs.
 
-#### Define alert conditions
+### Define alert conditions
+
+You can define alert conditions to ensure your test does not trigger in case of sporadic network blip and that you only get alerted in case of real issues with your endpoint.
```

Comment:
```suggestion
You can define alert conditions to ensure your test does not trigger for things like a sporadic network blip, so that you only get alerted in case of real issues with your endpoint.
```